### PR TITLE
add recaptcha to docker files

### PR DESCRIPTION
**Pull request by nllong**
#### Any background context you want to provide?
The recaptcha key is not passed when using docker-compose yml or local.

#### What's this PR do?
Add env vars to the docker comose files.

#### How should this be manually tested?
* export the recaptcha key to an env var
* docker-compose up --build

#### What are the relevant tickets?
#2882

### CODE REVIEW

File: seed/landing/views.py [hunk 1]
```diff
@@ -8,7 +8,6 @@
 import json
 import logging
 
-import botocore.exceptions
```
Comment (macintoshpie) [inline seed/landing/views.py]: not all seed builds have botocore 😬

Comment (nllong) [reply seed/landing/views.py]: oof, bad assumption on my end. Thanks for updating the message.

**Comment by macintoshpie** [comment]
@nllong I wasn't able to reproduce that templating error we were seeing on your end. Could you pull this and try again?

**Comment by macintoshpie** [comment]
@nllong any update on this?

**Comment by nllong** [comment]
> @nllong any update on this?

Okay, this works when starting with a clean local_untracked file. The issue on my local machine was that I kept the AWS_* variables uncommented which causes some issues. This can get merged after the tests pass! Thanks for the help on this one!

**Review by macintoshpie — APPROVED** [review]
🐳

**Pull request merged**